client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useStocktakeLineEdit.ts

```diff
@@ -48,10 +45,7 @@ export const useStocktakeLineEdit = (
 
   const addLine = () => {
     if (item) {
-      setDraftLines(lines => [
-        DraftLine.fromItem(id, item, defaultPackSize),
-        ...lines,
-      ]);
+      setDraftLines(lines => [DraftLine.fromItem(id, item), ...lines]);
```

Comment:
Went one step further - seeing as we're already passing in `item` - just using it directly in `DraftLine.fromItem` rather than passing in separately